samples/js/browser/avatar/js/chat.js

```diff
@@ -88,18 +88,22 @@ function connectAvatar() {
         messageInitiated = true
     }
 
-    const iceServerUrl = document.getElementById('iceServerUrl').value
-    const iceServerUsername = document.getElementById('iceServerUsername').value
-    const iceServerCredential = document.getElementById('iceServerCredential').value
-    if (iceServerUrl === '' || iceServerUsername === '' || iceServerCredential === '') {
-        alert('Please fill in the ICE server URL, username and credential.')
-        return
-    }
-
     document.getElementById('startSession').disabled = true
     document.getElementById('configuration').hidden = true
 
-    setupWebRTC(iceServerUrl, iceServerUsername, iceServerCredential)
+    const xhr = new XMLHttpRequest()
+    xhr.open("GET", `https://${cogSvcRegion}.tts.speech.microsoft.com/cognitiveservices/avatar/relay/token/v1`)
```

Comment:
You need consider private endpoint.
When user applies private endpoint, this URL should be like below:

https://{private endpoint}/tts/cognitiveservices/avatar/relay/token/v1